goth/node.py

```diff
@@ -21,7 +21,7 @@ def node_environment(
 
     daemon_env = {
         "CENTRAL_NET_HOST": f"{ROUTER_HOST}:{ROUTER_PORT}",
-        "ERC20_RINKEBY_GETH_ADDR": "http://ethereum:8545",
```

Comment:
I think we can keep this variable for compatibility with the latest stable `yagna` release. A comment along the lines of "Required for yagna prior to: https://github.com/golemfactory/yagna/pull/1208" should be enough to remove it in the future.

Reply:
Great idea! fixed :)